pom.xml

```diff
@@ -262,7 +262,7 @@
     <!-- Important: Selenium version 2.46.0 is the last one supported on Java 6! -->
     <version.org.jboss.arquillian.selenium>2.46.0</version.org.jboss.arquillian.selenium>
     <version.org.jboss.as.arquillian>7.2.0.Final</version.org.jboss.as.arquillian>
-    <version.org.jboss.as>7.5.4.Final-redhat-4</version.org.jboss.as>
+    <version.org.jboss.as>7.5.6.Final-redhat-2</version.org.jboss.as>
```

Comment:
Just checked locally and we can't easily upgrade these, since some of the artifacts are not published. Just one example `org.jboss.as:jboss-as-arquillian-container-managed:jar:7.5.6.Final-redhat-2` is not in maven.repository.redhat.com/ga